src/ui/Backdrop/Backdrop.ts

```diff
@@ -0,0 +1,74 @@
+import {Component} from '../Base/Component';
+import {ComponentOptions} from '../Base/ComponentOptions';
+import {IComponentBindings} from '../Base/ComponentBindings';
+import {Initialization, IInitializationParameters} from '../Base/Initialization';
+import {IResultsComponentBindings} from '../Base/ResultsComponentBindings';
+import {IQueryResult} from '../../rest/QueryResult';
+
+export interface IBackdropOptions {
+  imageField: string;
+  overlayColor?: string;
+  overlayGradient?: boolean;
+}
+
+/**
+ * This component is used to render an image URL (contained in a result's field)
```

Comment:
What happens if I use this component outside of a Card layout/ card template ?

Does it work ? Does it make sense to use it outside of a card template ?  Do we want to support this ?

If not, we should have a comment explaining this.